pyproject.toml

```diff
@@ -32,6 +32,7 @@ strict-rfc3339 = "^0.7"
 rfc3987 = "^1.3.8"
```

Comment:
Is the `strict-rfc3339` package still used somewhere?

Reply:
@tdilauro, I removed it.